docs/sources/get-started/ride-share-tutorial.md

```diff
@@ -0,0 +1,274 @@
+{{< /admonition >}}
+
+<!-- INTERACTIVE ignore END -->
+
+## Background
+
+In this tutorial, you will profile a simple "Ride Share" application. The application is a Python Flask app that simulates a ride-sharing service. The app has three endpoints which are found in the `server.py` file:
+
+- `/bike`    : calls the `order_bike(search_radius)` function to order a bike
+- `/car`     : calls the `order_car(search_radius)` function to order a car
+- `/scooter` : calls the `order_scooter(search_radius)` function to order a scooter
+
+To simulate a highly available and distributed system, the app is deployed on three distinct servers in 3 different regions: 
+- us-east
+- eu-north
+- ap-south
+
+This is simulated by running three instances of the server in Docker containers. Each server instance is tagged with the region it represents.
+
+{{< figure max-width="100%" src="/media/docs/pyroscope/ride-share-demo.gif" caption="Getting started sample application" alt="Getting started sample application" >}}
+
+In this scenario, a load generator will send mock-load to the three servers as well as their respective endpoints. This lets you see how the application performs per region and per vehicle type.
+
+{{<docs/ignore>}}
+{{< admonition type="tip" >}}
+A setup script runs in the background to install the necessary dependencies. This should take no longer than 30 seconds. Your instance will be ready to use once you `Setup complete. You may now begin the tutorial`.
+{{< /admonition >}}
+{{</docs/ignore>}}
+
+<!-- INTERACTIVE page intro.md END -->
+
+<!-- INTERACTIVE page step1.md START -->
+
+## Clone the repository
+
+1. Clone the repository to your local machine:
+
+    ```bash
+    git clone https://github.com/grafana/pyroscope.git && cd pyroscope
+    ```
+
+1. Navigate to the tutorial directory:
+
+    ```bash
+    cd examples/language-sdk-instrumentation/python/rideshare/flask
+    ```
+
+## Start the application
+
+Start the application using Docker Compose:
+
+```bash
+docker compose up -d
+```
+
+This may take a few minutes to download the required images and build the demo application. Once ready, you will see the following output:
+
+```console
+ ✔ Network flask_default  Created
+ ✔ Container flask-ap-south-1  Started
+ ✔ Container flask-grafana-1  Started
+ ✔ Container flask-pyroscope-1  Started     
+ ✔ Container flask-load-generator-1 Started 
+ ✔ Container flask-eu-north-1 Started       
+ ✔ Container flask-us-east-1 Started  
+```
+
+Optional: To verify the containers are running, run:
+
+```bash
+docker ps -a
+```
+<!-- INTERACTIVE page step1.md END -->
+
+<!-- INTERACTIVE page step2.md START -->
+
+## Accessing the Profile Explorer in Grafana
+
+Grafana includes a Profiles Explorer that you can use to view the profile data. To access the Profiles Explorer, open a browser and navigate to [http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer](http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer).
```

Comment:
Please note that the product name is Explore Profiles, just like Explore Traces, Explore Logs, and Explore Metrics.
```suggestion
## Accessing Explore Profiles in Grafana

Grafana includes the Explore Profiles app that you can use to view profile data. To access Explore Profiles, open a browser and navigate to [http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer](http://localhost:3000/a/grafana-pyroscope-app/profiles-explorer).
```

Reply:
Just thought about this - Does Killercoda handle our links that use GRAFANA_VERSION?